src/main/java/io/kokuwa/maven/helm/DependencyUpdateMojo.java

```diff
@@ -0,0 +1,58 @@
+package io.kokuwa.maven.helm;
+
+import java.nio.file.Path;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+import io.kokuwa.maven.helm.util.DependencyOverwriter;
+import lombok.Setter;
+
+/**
+ * Mojo for executing "helm dependency-update".
+ *
+ * @see <a href="https://helm.sh/docs/helm/helm_dependency_update/">helm dependency-update</a>
+ * @since 6.10.2
+ */
+@Mojo(name = "dependency-update", defaultPhase = LifecyclePhase.PROCESS_RESOURCES, threadSafe = true)
+@Setter
+public class DependencyUpdateMojo extends AbstractHelmMojo {
+
+	/**
+	 * Set this to <code>true</code> to skip invoking dependency-update goal.
+	 *
+	 * @since 6.10.2
+	 */
+	@Parameter(property = "helm.dependency-update.skip", defaultValue = "true")
+	private boolean skipDependencyUpdate;
+
+	@Override
+	public void execute() throws MojoExecutionException {
+
+		if (skip || skipDependencyUpdate) {
+			getLog().info("Skip dependency update");
+			return;
+		}
+
+		for (Path chartDirectory : getChartDirectories()) {
+
+			if (overwriteLocalDependencies) {
```

Comment:
A: Seems like we should move this somewhere common so we can have code reuse between this mojo and `DependencyBuildMojo`

Reply:
Moved to main abstract class